### PR TITLE
add mRNA-Seq part slides for plant bulk RNA-Seq tutorial

### PR DESCRIPTION
**Pull request by pavanvidem**
speaker notes are also available

### CODE REVIEW

**Comment by gallardoalba** [comment]
> speaker notes are also available

Thanks! I'll add my part.

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "Whole transcriptome analysis of arabidopsis Thaliana"
```
Comment (beatrizserrano) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:5]:
```suggestion
title: "Whole transcriptome analysis of Arabidopsis thaliana"
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+# (m)RNA-Seq data analysis
+
+???
+- (m)RNA-Seq data analysis
+
+---
+
+# Experimental design
+
+![Experimental deign](../images/plant_rnaseq_pipeline2.png)
```
Comment (beatrizserrano) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:26]:
```suggestion
![Experimental design](../images/plant_rnaseq_pipeline2.png)
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+.footnote[[*Oh E, Zhu JY, Bai MY, Arenhart RA et al., Elife, 2014*](https://www.ncbi.nlm.nih.gov/pubmed/24867218)]
+
+---
+
+## Tools used in mRNA-Seq data analysis
+
+![Tools used for the analysis](../images/tools_plant_mrna.png)
+
+???
+- We start with the quality assessment of the raw sequencing reads using Fast QC and Multi QC.
```
Comment (beatrizserrano) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:59]: FastQC and MultiQC?

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+- Then, we quantify the mRNAs using Salmon followed by the differential expression analysis using DE Seq 2.
```
Comment (beatrizserrano) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:60]: also DESeq2?

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+- Hence, if there is no significant amount of adapters present in the reads, adapter clipping can safely be skipped.
+- However, it is always good to trim the low-quality reads.
+- Salmon provides fast and accurate quantification.
+- Because of the large memory footprints of hash tables and suffix arrays, generally, transcriptome FASTA file is used for Salmon quantification instead of whole genome FASTA file.
+- Therefore it is advisable to use this tool on well-annotated organisms.
+- For the same reason, Salmon cannot be used to detect or annotate novel transcripts.
+
+---
+
+### plant small RNA target prediction using TargetFinder
```
Comment (beatrizserrano) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:141]:
```suggestion
### Plant small RNA target prediction using TargetFinder
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+- Penalty of 1 for mismatches, single-nucleotide gaps, or single-nucleotide bulges
+- Penalty of 0.5 G U base pairs
+- Doubled penalty scores at positions 2-13 relative to the 5' end of the small RNA query sequence
+- Duplexes with the following characteristics are rejected:
+    - more than one single-nucleotide bulge or gap
+    - more than seven total mismatches, G:U base pairs, bulges and gaps
+    - more than four total mismatches or four total G:U base pairs
+
+???
+- In animals usually, base pairing in the seed region of the micro RNA, that is 2 to 8 bases is enough for target recognition.
```
Comment (beatrizserrano) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:156]:
```suggestion
- In animals, usually, base pairing in the seed region of the micro RNA, that is 2 to 8 bases, is enough for target recognition.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html (hunk 3 above)
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:59]:
```suggestion
- We start with the quality assessment of the raw sequencing reads using FastQC and MultiQC.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html (hunk 4 above)
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:60]:
```suggestion
- Then, we quantify the mRNAs using Salmon followed by the differential expression analysis using DESeq2.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+---
+
+# Experimental design
+
+![Experimental deign](../images/plant_rnaseq_pipeline2.png)
+
+???
+- The second part of the tutorial deals with the mRNA-Seq data analysis and micro RNA target prediction.
+- First, we assess the quality of the mRNA-Seq reads, then quantify the genes, and carry out differential gene expression analysis.
+- Finally, we will predict which mRNAs are possibly downregulated by the upregulated micro RNAs.
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:31]:
```suggestion
- Finally, we will predict which mRNAs are possibly downregulated by the upregulated microRNAs.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html (hunk 3 above)
Comment (pavanvidem) [reply topics/transcriptomics/tutorials/mirna-target-finder/slides.html:59]: I don't know how Polly reads it. This is why I wrote them separated.

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+---
+
+# Background on data and tools
+
+???
+- Now we will look at  the data and tools that we are going to use in the tutorial
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:38]:
```suggestion
- Now we will look at the data and tools that we are going to use in the tutorial
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+
+---
+
+## Background on mRNA-Seq data
+
+![mRNA-Seq data used in the analysis](../images/data_plant_mrna.png)
+
+???
+- The mRNA-Seq data were generated using Illumina HiSeq 2000 sequencing platform.
+- Similar to micro RNA-Seq data, we have data from two conditions, namely control and brassinolide treatment.
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:48]:
```suggestion
- Similar to miRNA-Seq data, we have data from two conditions, namely control and brassinolide treatment.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+---
+
+## Tools used in mRNA-Seq data analysis
+
+![Tools used for the analysis](../images/tools_plant_mrna.png)
+
+???
+- We start with the quality assessment of the raw sequencing reads using Fast QC and Multi QC.
+- Then, we quantify the mRNAs using Salmon followed by the differential expression analysis using DE Seq 2.
+- Finally, we will use the Target Finder tool for micro RNA target prediction.
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:61]:
```suggestion
- Finally, we will use the TargetFinder tool for miRNA target prediction.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+- Penalty of 0.5 G U base pairs
+- Doubled penalty scores at positions 2-13 relative to the 5' end of the small RNA query sequence
+- Duplexes with the following characteristics are rejected:
+    - more than one single-nucleotide bulge or gap
+    - more than seven total mismatches, G:U base pairs, bulges and gaps
+    - more than four total mismatches or four total G:U base pairs
+
+???
+- In animals usually, base pairing in the seed region of the micro RNA, that is 2 to 8 bases is enough for target recognition.
+- Plant micro RNAs require more stringent base pairing. Generally, a near-perfect pairing in 5 prime region and a substantial pairing in 3 prime region is necessary.
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:157]:
```suggestion
- Plant miRNAs require more stringent base pairing. Generally, a near-perfect pairing in 5 prime region and a substantial pairing in 3 prime region is necessary.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+- Due to this strict pairing mechanism, plant miRNAs have a significantly less number of target genes.
+- For this reason, we use a plant-specific micro RNA target prediction algorithm called Target finder.
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:159]:
```suggestion
- For this reason, we use a plant-specific miRNA target prediction algorithm called TargetFinder.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+- Target finder takes a small RNA sequence and a FASTA formatted file of the target sequence database.
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/mirna-target-finder/slides.html:160]:
```suggestion
- TargetFinder takes a small RNA sequence and a FASTA formatted file of the target sequence database.
```

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html (hunk 3 above)
Comment (gallardoalba) [reply topics/transcriptomics/tutorials/mirna-target-finder/slides.html:59]: Then those changes can be included after the recording :)

Comment (beatrizserrano) [reply topics/transcriptomics/tutorials/mirna-target-finder/slides.html:59]: ah true, that makes sense, ignore my comments then and let's make Polly happy :)

**Comment by bgruening** [comment]
> > speaker notes are also available
> 
> Thanks! I'll add my part.

@gallardoalba is your part ready? Can we proceed here?

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html (hunk 11 above)
Comment (hexylena) [reply topics/transcriptomics/tutorials/mirna-target-finder/slides.html:228]: could just be added to the ari-map.yml

**Comment by gallardoalba** [comment]
> > > speaker notes are also available
> > 
> > 
> > Thanks! I'll add my part.
> 
> @gallardoalba is your part ready? Can we proceed here?

Yes, now it is ready.

**Review by gallardoalba — APPROVED** [review]

File: topics/transcriptomics/tutorials/mirna-target-finder/slides.html (hunk 11 above)
Comment (pavanvidem) [reply topics/transcriptomics/tutorials/mirna-target-finder/slides.html:228]: do I have to add definitions for both the `miRNA` and `MiRNA` or is it case insensitive?

Comment (hexylena) [reply topics/transcriptomics/tutorials/mirna-target-finder/slides.html]: It should be insensitive

**Review by hexylena — APPROVED** [review]
This looks great, thanks everyone!

**Comment by hexylena** [comment]
The video building will go around ~14:03, be sure to check back then to see your video!

**Comment by pavanvidem** [comment]
hi @hexylena there is a blunder. the links to the images are not correct. sorry! I will correct them.

**Comment by hexylena** [comment]
No worries, I should've caught it.

**Comment by hexylena** [comment]
It's live! https://training.galaxyproject.org/training-material/videos/watch.html?v=transcriptomics/tutorials/mirna-target-finder/slides can you please check the pronunciation everywhere and correct anything that's too wrong? Please let me know if you need suggestions for how to write it to get it to pronounce it properly, I can test phrases locally.

**Pull request merged**